project/settings.sc

```diff
@@ -816,7 +816,7 @@ trait ScalaCliScalafixModule extends ScalafixModule {
   }
 }
 
-trait ScalaCliCrossSbtModule extends CrossSbtModule with ScalaCliModule
+trait ScalaCliCrossSbtModule extends Cross.Module[String] with CrossSbtModule with ScalaCliModule
```

Comment:
This is not needed since `CrossSbtModule` extends `CrossModuleBase` which in turn extends has `Cross.Module[String]` mixed-in.